src/docs/deploy/builds.md

```diff
@@ -111,7 +111,7 @@ _Note, negations will only work if you include files in a preceding rule._
 
 <PriorityBoardingBanner />
 
-With Nixpacks, we analyze the app source directory and generate a build plan. This determines which language provider to install packages and runtimes for.
+With Nixpacks, we analyze the app source directory and generate a build plan. This determines which language provider to install and the required packages and runtimes.
```

Comment:
Loving this tool even more since it's flagging this.  Although small nit, not sure about the suggested fix.. I don't think it reads like it's supposed to (yes using the same error in my comment lol).  Curious about your thoughts.